CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added proper go linter
 - Fixed DNS proxy upstream lookup loop
 - Updated the documentation to reflect the changes regarding IPv6
+- Added Mergeable config
```

Comment:
The above points are no longer  "Latest" but part of the releases 0.5.0-0.5.2. The section needs renaming and this entry should be moved into a new "Latest" section.